internal/signature/sign.go

```diff
@@ -209,3 +242,39 @@ func matchSAN(cert *x509.Certificate, name, email string) bool {
 
 	return false
 }
+
+func attachRekorLogEntry(ctx context.Context, sd *cms.SignedData, cert *x509.Certificate, addr string) (*models.LogEntryAnon, error) {
+	rekor, err := rekor.New(addr)
```

Comment:
Would it be easier to pass in the Rekor client directly? I always prefer to dependency-inject clients.

Reply:
You already have a rekor.Client in git.Sign.